aliyun-ecs/src/main/java/org/jclouds/aliyun/ecs/domain/options/TagOptions.java

```diff
@@ -84,7 +84,7 @@ public static TagOptions keys(Set<String> keys) {
    private void validateInput(final String input, int maxLength) {
       checkNotNull(input);
       checkState(input.length() <= maxLength, String.format("input must be <= %d chars", maxLength));
-      checkState(Iterables.any(FORBIDDEN_PREFIX, new Predicate<String>() {
+      checkState(Iterables.contains(FORBIDDEN_PREFIX, new Predicate<String>() {
```

Comment:
Hmmmm If I understand this correctly, it is still not right.
You want to check that the given input **does not** contain any of the forbidden inputs, right? It should be something like>
```java
checkState(!Iterables.any(FORBIDDEN_PREFIX, new Predicate<String>() {
```